bfd/elfxx-riscv.c

```diff
@@ -2622,6 +2625,8 @@ riscv_multi_subset_supports_ext (riscv_parse_subset_t *rps,
       return "xcorevmem' or `xcorev";
     case INSN_CLASS_COREV_BI:
       return "xcorevbi' or `xcorev";
+    case INSN_CLASS_COREV_ELW:
+      return "xcorevelw' or `xcorev";
```

Comment:
TBD - what does `corev` mean when we have new versions of the tool chain. Suggestion
- `corev1p0`: Version 1 of all 5 CV32E40Pv1 extensions
- `corev2p0`: Version 2 of all 5 CV32E40Pv1 extensions and version ` of all C32E40Pv2 extensions